src/main/java/com/monumental/controllers/TagController.java

```diff
@@ -30,14 +32,19 @@
     public List<Tag> getTags(@RequestParam(required = false) Integer monumentId,
                              @RequestParam(required = false) String name,
                              @RequestParam(required = false) List<String> names) {
+        List<Tag> tags;
+
         if (monumentId != null) {
-            return this.tagRepository.getAllByMonumentId(monumentId);
+            tags = this.tagRepository.getAllByMonumentId(monumentId);
         } else if (name != null) {
-            return this.tagRepository.getAllByName(name);
+            tags = this.tagRepository.getAllByName(name);
         } else if (names != null && names.size() > 0) {
-            return this.tagRepository.getAllByNameIn(names);
+            tags = this.tagRepository.getAllByNameIn(names);
         } else {
-            return this.tagRepository.findAll();
+            tags = this.tagRepository.findAll();
         }
+
+        tags.sort(Comparator.comparing(Tag::getName));
```

Comment:
Java 8 has some neat stuff like this.

Reply:
I'm trying to decide if this is better or worse than the Javascript equivalent:
```
tags.sort((a, b) => b.getName() - a.getName());
// or if sorting an array of strings
tags.sort();
```